### ISSUE TITLE
BingMapsRESTToolKit v1.1.4 is not finding valid addresses: We use this to find the distance between two valid waypoints for mileage calculations.

### ISSUE THREAD

**Issue by shadowDragon2022** [microsoft/BingMapsRESTToolkit#71]
- Customers have reported issues while calculating mileage between two waypoints.
– Investigating this issue we have tried changing the format and have mixed results and the calls we are getting for addresses not being found is growing. Below I will show waypoint calculations as well as just addresses not being found or has incorrect results.
– I do have more I can provide, but was not sure if more were needed to help resolve this.

Driving Distance Calculations:

Example One:
Starting Location:
3724 Singleton Terrace
Frederck MD 21704

Ending Location:
43 Degrange St
Frederick MD 21701

Error returned: One or more locations specified in the waypoint parameter are invalid or require more information “43 Degrange St, Fredrick MD 21701,”

Example Two:
Starting Location:
218 Tamms Rd
Middletown, NY 10941

Ending Location:
770 Route 17M
Middletown, NY 10940

Error returned: One or more locations specified in the waypoint parameter are invalid or require more information “770 Route 17M, NY 10940”

Address lookup:

Example One:

2528 Route 17M, Goshen, NY 10924

Geolocation changes the above address to:

2528 Route 17M, Harriman, NY 10926

**Comment by rbrundritt** [comment]
Can you please report this to the Bing Maps Enterprise support channel here: https://www.microsoft.com/en-us/maps/support 

This project is not a support channel.

**Comment by shadowDragon2022** [comment]
Yeah I just keep getting from one support channel to the next and everyone says go here and then when I do I get the above.  Super frustrating!

**Comment by shadowDragon2022** [comment]
The issues I have are with the SDK which is Bing.  So when the SDK is not parsing the map correctly the Bing Maps still is.  So I can't report to Bing Maps since it's not an issue and I can't report here because it's not a support channel and I can report in Visual Studio because they don't support this.  Is this a loop of not supporting your product?

**Comment by rbrundritt** [comment]
The Bing Maps Enterprise support team can handle any requests related to the Bing Maps Enterprise SDKs. This library is an open source project that wraps the SDKs and is not the SDK itself. I created this library 7 years ago and am not on the Bing Maps team.
